sklearn/metrics/_pairwise_distances_reduction.pyx

```diff
@@ -529,8 +529,10 @@ cdef class PairwiseDistancesArgKmin(PairwiseDistancesReduction):
         chunk_size=None,
         dict metric_kwargs=None,
         n_threads=None,
-    ) -> PairwiseDistancesArgKmin:
-        """Return the PairwiseDistancesArgKmin implementation for the given arguments.
+        str strategy=None,
+        bint return_distance=False,
+    ):
+        """Return the results of the reduction for the given arguments.
 
```

Comment:
```suggestion

    The public `compute` classmethod is responsible of introspecting the arguments
    values to dispatch to the `_compute` instance method of the most appropriate
    implementations.
    
    All temporarily allocated datastructures necessary for the concrete implementation
    are therefore freed when the classmethod returns.
```

Reply:
Introduced in the `Notes` section via 6983c3215497e8b9c1040ce49b2d736f32ef957d.